test/e2e/framework/kubelet/stats.go

```diff
@@ -548,14 +548,14 @@ func (r *ResourceMonitor) GetLatest() (ResourceUsagePerNode, error) {
 	return result, utilerrors.NewAggregate(errs)
 }
 
-// GetMasterNodeLatest returns the latest resource usage of master and node.
-func (r *ResourceMonitor) GetMasterNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
+// GetControlPlaneNodeLatest returns the latest resource usage of controlplane and node.
+func (r *ResourceMonitor) GetControlPlaneNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
 	result := make(ResourceUsagePerNode)
-	var masterUsage ResourceUsagePerContainer
+	var controlplaneUsage ResourceUsagePerContainer
 	var nodesUsage []ResourceUsagePerContainer
 	for node, usage := range usagePerNode {
-		if strings.HasSuffix(node, "master") {
-			masterUsage = usage
+		if strings.HasSuffix(node, "controlplane") {
```

Comment:
I _think_ that unless you update the possible string values for a node suffix to not have "master" be an option and instead have "controlplane", this condition will never be true. You either have to leave this as is. Or update possible string suffixes and all possible references to that suffix.